extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/java/org/apache/guacamole/auth/mysql/MySQLEnvironment.java

```diff
@@ -88,6 +88,12 @@
     private int DEFAULT_MAX_CONNECTIONS = 0;
 
     /**
+     * The default value for the connection weight for a connection in
+     * a balancing group.
+     */
+    private int DEFAULT_CONNECTION_WEIGHT = 0;
```

Comment:
Removed this completely - the DB-specific default connection weight was not being used at all.